### PR TITLE
Add coin to overload cowns in systematic testing

### PR DESCRIPTION
**Pull request by Theodus**
This PR adds a coin flip to artificially overload a running cown before each behaviour step.

### CODE REVIEW

File: src/rt/sched/status.h [hunk 1]
```diff
@@ -119,6 +119,15 @@ namespace verona::rt
       _load_hist |= (_current_load >> 4);
       _current_load = 0;
     }
+
+    /// Artificially overload a cown.
+    /// TODO: In the future, this will require a bit to remain overloaded after
+    /// the load drops.
+    inline void overload()
+    {
+      _load_hist = ~(uint32_t)0;
```
Comment (rengolin) [inline src/rt/sched/status.h:128]: -1?

**Comment by mjp41** [comment]
I don't see where the coin flip is?

**Comment by Theodus** [comment]
@mjp41, there was no coin flip. I had this PR set as a draft since I still had to get it working properly. I should have been more clear about that.

**Review by mjp41 — APPROVED** [review]
LGTM

**Pull request merged**